content/blog/why-is-redfish®-different-from-other-rest-apis-part-1.md

```diff
@@ -0,0 +1,134 @@
+
+![Figure1: Service-Client model](/img/fig-1-modelserviceclient.png "Figure1: Service-Client model")
+
+Redfish® clients are numerous and varied. For development and troubleshooting, the <a href="https://www.postman.com/" target="_blank">Postman API platform</a> is very popular. For one-off actions or “quick and dirty” scripts, <a href="https://curl.se/" target="_blank">cURL</a>, PowerShell, Python or even <a href="https://github.com/HewlettPackard/python-redfish-utility/releases/latest" target="blank">HPE iLOrest</a> do the trick. For more sophisticated client programs, you can use an <a href="https://galaxy.ansible.com/ui/repo/published/hpe/ilo/" target="_blank">Ansible playbook library</a>, Chef or Go and its <a href="https://pkg.go.dev/github.com/stmcginnis/gofish/redfish" target="_blank">Go-Redfish library</a>.
+
+Proprietary monitoring applications (i.e., <a href="https://www.hpe.com/us/en/hpe-greenlake-compute-ops-management.html" target="_blank">HPE Compute Ops Management</a>) or open source applications (<a href="https://github.com/ODIM-Project/ODIM" target="_blank">ODIM</a>, <a href="https://github.com/nsfcac/Nagios-Redfish-API-Integration" target="_blank">Nagios</a>, etc.) constitute native Redfish® clients or can become so using specific plugins.
+
+### Remote or local management
+
+In most cases, Redfish® clients performs <a href="https://en.wikipedia.org/wiki/Out-of-band_management" target="_blank">out-of-band management</a>. However, it is entirely possible to install the client in the OS of the server to be managed and to access the BMC via an internal path to the server (in-band-management). This path can be proprietary like HPE's CHIF (<a href="https://developer.hpe.com/blog/chif-driver-not-found/" target="_blank">Channel Interface</a>) or the <a href="https://en.wikipedia.org/wiki/Ethernet_over_USB" target="_blank">Ethernet over USB</a> protocol which allows an IP connection between the OS and the BMC.
+
+### Protocol and data model separation
+
+The Redfish® protocol specification is published in DMTF document <a href="https://www.dmtf.org/dsp/DSP0266" target="_blank">DSP0266</a> while data modeling in <a href="https://www.dmtf.org/dsp/DSP0268" target="_blank">DSP0268</a>. This separation allows flexibility with regard to the management of version numbers. Without this separation, a minor change to the protocol specifications (e.g. adding an acronym to the glossary) would involve an increment of the overall specification version number and impose a set of unnecessary tests to requalify clients.
```

Comment:
I think you're missing a word in the first sentence. It also needs a comma. I think it should read "...document DSP0266, while data modeling **is in** DSP0268."